tests/test_compression.py

```diff
@@ -1,16 +1,17 @@
 """
 Here we only test that reading and writing with compression keeps the data intact.
 """
+
 import pathlib
 
 import pytest
-from pandas._testing import assert_dict_equal
-from pandas.testing import assert_frame_equal
 
 from tfs.reader import read_headers, read_tfs
 from tfs.writer import write_tfs
 
-CURRENT_DIR = pathlib.Path(__file__).parent
+from .conftest import INPUTS_DIR, assert_tfs_frame_equal
+
+# ----- Compression tests with 'classic' TFS files (no MAD-NG features) ----- #
 
 
 @pytest.mark.parametrize("extension", ["gz", "bz2", "zip", "xz", "zst", "tar", "tar.gz"])
```

Comment:
make SUPPORTED_EXTENSIONS constant to be used in all `parametrize`

Reply:
Ok, but as a constant in the file itself only. We basically support what `pandas` supports and they might change that so I don't want us to hardcode it into the package.